src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -601,17 +591,6 @@ public void publishSignatureScanReport(List<SignatureScannerReport> report) thro
         });
     }
 
-    public SignatureScannerCodeLocationResult calculateWaitableSignatureScannerCodeLocations(
-        NotificationTaskRange notificationTaskRange,
-        List<SignatureScannerReport> reports
-    ) throws OperationException {
-        return auditLog.namedInternal(
-            "Calculate Signature Scanner Waitable Code Locations",
-            () -> new CalculateWaitableSignatureScanCodeLocations()
-                .calculateWaitableCodeLocations(notificationTaskRange, reports)
-        );
```

Comment:
The MaaS changes on branch sb-correlation-id-and-counts rely on the waitable codelocation list in CodeLocationAccumulator, so I think we need to preserve that or something equivalent so Detect can provide correlated scan counts at the end of the run for MaaS.